src/components/ruleOverview/ruleOverview.js

```diff
@@ -17,15 +17,15 @@ class RuleOverviewFlyout extends Component {
         };
```

Comment:
is this equivalent to:
```
this.state = {
  enabled: props.rules.some(rule => !rule.Enabled)
}
```

Reply:
No. If the rule is already disabled - then when we open the flyout, we see the enable toggle.
If the rule is already ENABLED -  then when we open the flyout , we see the disable toggle with out user toggles them.